content/docs/en/elements/components/list-view.md

```diff
@@ -4,53 +4,77 @@ apiRef: https://docs.nativescript.org/api-reference/classes/_ui_list_view_.listv
+When you create conditions for `<v-template>`, you can use any valid JavaScript expression or any of the following helpers:
+
+* `$index`
+* `$even` 
+* `$odd`
 
-To learn more about the `v-template` component, head over to the [`v-template` documentation](/en/docs/utilities/v-template).
+## Using `<ListView>` with `v-for`
 
-The ListView does not loop through the items as you would expect when using a [`v-for`](https://vuejs.org/v2/guide/list.html#Mapping-an-Array-to-Elements-with-v-for) loop. Instead it only creates the necessary views to display the currently visible items on the screen, and when scrolling it reuses the views that are already off-screen. This concept is called view recycling, and is commonly used in mobile applications to improve performance. This is important because you can't rely on event listeners attached inside the `v-template`, instead you should use the `itemTap` event which contains the index of the tapped item, as well as the actual item from the list.
+When you use `<ListView>` with [`v-for`](https://vuejs.org/v2/guide/list.html#Mapping-an-Array-to-Elements-with-v-for), you need to use the default `itemTap` event instead of attaching other event listeners. The `itemTap` event contains the index of the tapped item and the actual item from the list.
 
-```js
+```javascript
 onItemTap(event) {
   console.log(event.index)
   console.log(event.item)
 }
 ```
 
-[> screenshots for=ListView <]
-
 ## Props
 
-| name | type | description |
+| Name | Type | Description |
 |------|------|-------------|
-| `for` | `String` | The expression for iterating through the items, examples:<br>- `item in listOfItems`<br>- `(item, index) in listOfItems`<br>- `item in [1, 2, 3, 4, 5]`
-| `items` | `Array<any>` | An array of items to be displayed in the ListView. **Note**: This property is only for advanced use, we recommend using the `for` property in most cases.
-| `separatorColor` | `Color` | Set the separator line color. Set to `transparent` to remove it.
+| `for` | `String` | Provides the expression for iterating through the items.<br/>For example: <ul><li><code>item in listOfItems</code></li><li><code>(item, index) in listOfItems</code></li><li><code>item in [1, 2, 3, 4, 5]</code></li></ul>
+| `items` | `Array<any>` | An array of items to be shown in the `<ListView>`.<br/>**This property is only for advanced use. Use the `for` property instead.**
+| `separatorColor` | `Color` | Sets the separator line color. Set to `transparent` to remove it.
 
 ## Events
 
-| name | description |
+| Name | Description |
 |------|-------------|
-| `itemTap`| Emitted when an item in the ListView has been tapped. To access the tapped item use `event.item`.
+| `itemTap`| Emitted when an item in the `<ListView>` is tapped. To access the tapped item, use `event.item`.
+
+## Methods
+
+| Name | Description |
+|------|-------------|
+| `refresh()` | Forces the `<ListView>` to reload all its items.
```

Comment:
This method doesn't actually exist in this form, only on the underlying native element, however we can add it to make things easier, so keep it here, but maybe add a `(soon)` note next to it, I've opened the issue here to track progress
https://github.com/nativescript-vue/nativescript-vue/issues/193

Reply:
👍